### DISCUSSION TITLE
Welcome to iOSApp360 Discussions!

### DISCUSSION THREAD

**Discussion by sikshakosh** [wideclassrooms/iOSApp360#1] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KashanAhmad123** [comment]
Did I have to create the same page , and run on local host ?

**Reply by sikshakosh** [reply]
No, you have to create same screen using XML in iOS App. You can use the functions available in the project

**Comment by KashanAhmad123** [comment]
Okay , thank you for the clarification ,
Again I will ask you if require .

**Reply by sikshakosh** [reply]
Sure

**Reply by Preetamkumarchauasiya** [reply]
Did I have to create the iOSApp360 Login page?

**Reply by sikshakosh** [reply]
@Preetamkumarchauasiya No, you need to create same design using XML Template.

**Reply by Preetamkumarchauasiya** [reply]
Okay, I will create it.

**Comment by Preetamkumarchauasiya** [comment]
Did I have to create the iOSApp360 Login page?

**Comment by KashanAhmad123** [comment]
Yes, Mam please elaborate this swift session .

**Comment by KashanAhmad123** [comment]
I appreciate your response , but I will try to create by myself and if it
getting extreme rigid I will cam to you for help.

**Comment by KashanAhmad123** [comment]
By the if we got connected on any social media platform then it will be
better.

**Reply by sikshakosh** [reply]
Thanks for interest. Pls [Click](https://www.linkedin.com/in/wideclassrooms-learning-4569551a0l) to connect on LinkedIn.
To complete task you have to create custom view classes according to XML tags then parse XML to Class objects.

**Comment by KashanAhmad123** [comment]
Your LinkedIn profile is not opening , there is pop up coming which tell
this profile is not available.

**Reply by sikshakosh** [reply]
You can find us with name WideClassrooms on LinkedIn.
